mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/proxy/MQTTProxyConfiguration.java

```diff
@@ -131,6 +131,13 @@
     )
     private String defaultNamespace = "default";
 
+    @FieldContext(
+            category = CATEGORY_MQTT,
+            required = true,
+            doc = "Default Pulsar topic that the MQTT server used."
```

Comment:
Default Pulsar topic that the MQTT server used -> Default Pulsar topic domain the MQTT server used.